x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/form/field_config.tsx

```diff
@@ -895,7 +895,27 @@ export const FIELD = (readOnly?: boolean): FieldMap => ({
       isEditFlow: isEdit,
     }),
     validation: () => ({
-      validate: (value) => Boolean(value.script),
+      validate: (value) => {
+        // return false if script contains import or require statement
+        if (
+          value.script?.includes('import ') ||
+          value.script?.includes('require(') ||
```

Comment:
A `regex` or other method might be better here, as this approach is somewhat naive. `require (` gets around it. How much of an issue do we think this will be?

```typescript
const requireRegex = new RegExp(/require\s*\(/);
if (requireRegex.test(value.script)) {}
```

Reply:
i think it's a really simple validation. includes works perfectly IMO. using regex will complicate it.

Reply:
Ok - after offline discussion we are only intending this validation to catch the basic use case that a user has copied a UJ script in here without modifying any of the require/import/journey code in a way that would defeat this simpler check. Comprehensive catching of these keywords is not the target of this code.